initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -863,10 +863,17 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
-            <xs:attribute name="memCachedOptionString" type="xs:string" use="optional">
+            <xs:attribute name="ensureESPCache" type="xs:boolean" use="optional" default="true">
```

Comment:
If an existing ESP is upgraded, doesn't have added config, and doesn't have memcached running, it will not work. ESP log will show an error. The intention is to encourage ESP cache. But, I have already heard the complaint about that.